CommonMark/CommonMarkException.cs

```diff
@@ -8,7 +8,7 @@ namespace CommonMark
     /// <summary>
     /// An exception that is caught during CommonMark parsing or formatting.
     /// </summary>
-#if v2_0 || v3_5 || v4_0 || v4_5
+#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
```

Comment:
Would something like this be better?

https://github.com/AMDL/CommonMark.NET/commit/bcffce97662fba96742ca5fbac1de20d25c2c427